SConstruct

```diff
@@ -447,6 +447,8 @@ env_base["debug_symbols"] = methods.get_cmdline_bool("debug_symbols", env_base.d
 
 if env_base.editor_build:
     env_base.Append(CPPDEFINES=["TOOLS_ENABLED"])
+    if env_base["engine_update_check"]:
+        env_base.Append(CPPDEFINES=["ENGINE_UPDATE_CHECK_ENABLED"])
```

Comment:
I'd prefer to keep it where it was, but adding `env_base.editor_build and ...` there.

The defines around this line are the very core definitions of what "editor build" means, while the update check stuff is really not important and can be handled later (I mean for the reader, doesn't make a difference for the buildsystem of course).